pkg/ipc/ipc.go

```diff
@@ -504,9 +505,10 @@ const (
 )
 
 type handshakeReq struct {
-	magic uint64
-	flags uint64 // env flags
-	pid   uint64
+	magic      uint64
+	flags      uint64 // env flags
+	pid        uint64
+	sandboxArg int
```

Comment:
`int` type does not have fixed binary representation, so passing it between programs is fragile. Say, if fuzzer is build for amd64 and executor for 386 arch. That's the reason we use `uint64` for all fields. Please make it uint64 as well.